src/nyc_trees/js/src/reserveBlockfacePage.js

```diff
@@ -13,3 +23,35 @@ var reservationMap = MapModule.create({
 L.tileLayer(config.urls.layers.reservable.tiles, {
     maxZoom: zoom.MAX
 }).addTo(reservationMap);
+
+var $current = $(dom.currentReservations),
+    selectedBlockfacesCount = +($current.text()),
+    blockfaceLimit = +($(dom.totalReservations).text()),
+    selectedBlockfaces = {};
+
+selectableBlockfaceLayer.create({
+    map: reservationMap,
+    gridUrl: config.urls.layers.reservable.grids,
+
+    onAdd: function(gridData) {
+        if (selectedBlockfacesCount < blockfaceLimit && gridData.restriction === 'none') {
+            selectedBlockfaces[gridData.id] = gridData;
+            selectedBlockfacesCount++;
+            $current.text(selectedBlockfacesCount);
+
+            return true;
```

Comment:
Blockface selection appeared to suddenly stop working after I had chosen a few. It took me a minute to realize that I had clicked on the maximum number of blockfaces that I was allowed to reserve. Consider some kind of notification message for all cases that cause `onAdd` to return false.